### PR TITLE
Metric test

### PR DESCRIPTION
**Pull request by CorneliaKelinske**
Final review please and thank you!

### CODE REVIEW

File: test/graphql_practice_web/schema/mutations/upload_test.exs [hunk 1]
```diff
@@ -21,152 +21,152 @@ defmodule GraphqlPracticeWeb.Schema.Mutations.UploadTest do
...
-
-      assert %{"deleteUpload" => deleted_upload} = data
-
-      assert %{"description" => "some description", "id" => ^string_id, "title" => "some title"} =
-               deleted_upload
-
-      assert_raise Ecto.NoResultsError, fn -> Content.get_upload!(id) end
-    end
-  end
+  # describe "@create_upload" do
```
Comment (alanvardy) [inline test/graphql_practice_web/schema/mutations/upload_test.exs]: Why is this commented out?

File: test/graphql_practice_web/schema/queries/metric_test.exs [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+      assert {:ok, %{data: data}} = Absinthe.run(@users_doc, Schema)
+      Absinthe.run(@users_doc, Schema)
+
+      assert {:ok, %{data: data}} =
+               Absinthe.run(@count_request_doc, Schema, variables: %{"request" => "users"})
+
+      assert %{"countRequest" => %{"count" => 2, "request" => "users"}} = data
+    end
+  end
+end
```
Comment (alanvardy) [inline test/graphql_practice_web/schema/queries/metric_test.exs:55]: You could also make a test module for the metric specifically, that does not go through GQL if you wish. I think for the puposes of this exercise the above is sufficient though.

**Review by alanvardy — APPROVED** [review]
Confused by the commented out tests but otherwise good.

File: test/graphql_practice_web/schema/mutations/upload_test.exs (hunk 1 above)
Comment (CorneliaKelinske) [reply test/graphql_practice_web/schema/mutations/upload_test.exs]: Ahhhh! Because I forgot that they were commented out.....

**Pull request merged**